src/v0/destinations/mp/transform.js

```diff
@@ -182,11 +182,11 @@ const processTrack = (message, destination) => {
 };
 
 const processIdentifyEvents = async (message, type, destination) => {
-  let returnValue;
+  const returnValue = [];
 
-  // Creating the response to identify an user
+  // Creating the user profile
   // https://developer.mixpanel.com/reference/profile-set
-  returnValue = createIdentifyResponse(message, type, destination, responseBuilderSimple);
+  returnValue.push(createIdentifyResponse(message, type, destination, responseBuilderSimple));
```

Comment:
Earlier, an external Axios call was made to create a user, which is causing the issue because we haven't applied any batching for the create user operation. Now, we are multiplexing these calls and sending them in the same order (create user first, then merge second) within a batch.

Reply:
Earlier, an external Axios call was made to create a user, which is causing the issue because we haven't applied any batching for the create user operation. Now, we are multiplexing these calls and sending them in the same order (create user first, then merge second) within a batch. 
cc- @krishna2020